### DISCUSSION TITLE
No fan contol for Legion 5i Pro (2022)

### DISCUSSION THREAD

**Discussion by pauljvallance** [BartoszCichecki/LenovoLegionToolkit#921] [Q&A]
Hi,
I cannot find any fan control in the latest Legion Toolkit on Window 11, though all my websearches say that toolkit  fan control works with this model :

Model - Legion 5 Pro 16IAH7H
Bios version - J2CN51WW
Model Type - 82RF

**Comment by BartoszCichecki** [comment]
As written in the readme, fan control is possible in `Custom` Power Mode.

**Comment by pauljvallance** [comment]
Thanks for the RTFM snub!

So where is the readme - not in the install folder, nor in the Toolkit dashboard either as far as I can see? I can’t find it anywhere. Perhaps you could make its location a little more obvious and accessible before you get all snarky with me.

May I suggest you add a link to any documentation you have in the ‘about' tab of the toolbar down the left hand side of the dashboard.

If I had found the documentation in the first place  I wouldn’t be troubling you.

Paul





--
Paul Vallance

tel: +370 690 22242
email: ***@***.***
skype: pauljvallance

**Reply by BartoszCichecki** [reply]
It's on the main project page: https://github.com/BartoszCichecki/LenovoLegionToolkit just scroll down a bit...
